### PR TITLE
Update import dependencies

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Fixing Dependency Bot alerts

Screen Shot 2021-09-10 at 9 08 38 AM

#### Ticket Link
NONE

### CODE REVIEW

**Comment by sbishel** [comment]
Hey @chenilim, I meant to do this earlier in the week, but I'm thinking we should CP to 0.9.

**Comment by chenilim** [comment]
No worries @sbishel. It looks like these only affect the import scripts, so won't affect the v0.9.0 release one way or another. I don't think it's strictly needed, but go ahead and cherry-pick to `release-0.9.0` if you think that's a good idea.

**Review by harshilsharma63 — APPROVED** [review]

**Review by chenilim — APPROVED** [review]
Wasn't sure if you were still waiting for my review, but LGTM!

**Pull request merged**
